### PR TITLE
Add search bar

### PR DESCRIPTION
**Pull request by CathrinTruchan**
Hello :) 
I added a search bar to the index and the flow-page. I am looking forward to your feedback. Thanks :)

### CODE REVIEW

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey, I added some suggestions, I think you made it yourself much harder than it needed to be

File: pages/_app.js [hunk 1]
```diff
@@ -2,10 +2,16 @@ import GlobalStyle from "../components/GlobalStyle";
...
 function MyApp({ Component, pageProps }) {
+  const [searchQuery, setSearchQuery] = useState("");
+
   return (
     <>
       <GlobalStyle />
-      <Component {...pageProps} />
+      <Component
+        {...pageProps}
+        searchQuery={searchQuery}
```
[the remark this answers is not in the document]
Comment (CathrinTruchan) [reply pages/_app.js:12]: Ah, of course. That's easier :) Thanks!

**Comment by CathrinTruchan** [comment]
> Hey, I added some suggestions, I think you made it yourself much harder than it needed to be

Thanks a lot for reviewing @F-Kirchhoff! I added the changes you requested. Can you check the code again?

File: pages/index.js [hunk 2]
```diff
@@ -17,10 +19,13 @@ export default function Home({ searchQuery, setSearchQuery }) {
...
-          {asanas.map((asana) => {
-            const nameInLowerCase = asana.english_name.toLowerCase();
-            const searchQueryInLowerCase = searchQuery.toLowerCase();
-            if (nameInLowerCase.includes(searchQueryInLowerCase)) {
+          {asanas
+            .filter((asana) => {
+              const nameInLowerCase = asana.english_name.toLowerCase();
+              const searchQueryInLowerCase = searchQuery.toLowerCase();
+              return nameInLowerCase.includes(searchQueryInLowerCase);
+            })
```
Comment (F-Kirchhoff) [inline pages/index.js:27]: nice!

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🦊

**Comment by CathrinTruchan** [comment]
> LGTM 🦊

Thanks ✌️

**Pull request merged**